versions.json

```diff
@@ -0,0 +1,12 @@
+{
+    "api": "0",
+    "distros": {
+        "cro": "0.8.10",
+        "Cro::Core": "0.8.10",
+        "Cro::HTTP": "0.8.10",
+        "Cro::TLS": "0.8.10",
+        "Cro::WebApp": "0.9.0",
+        "Cro::WebSocket": "0.8.10"
+    },
+    "docker": "0.8.10"
```

Comment:
I wouldn't say "docker" here, but rather the more generic "container_image" or perhaps the more precise "oci_image".